modules/govuk/manifests/apps/email_alert_service.pp

```diff
@@ -65,6 +71,20 @@
     process_regex  => '\/rake message_queues:unpublishing_consumer',
   }
 
+  govuk::procfile::worker { 'email-alert-service-subscriber-list-details-update-major-consumer':
+    setenv_as      => $app_name,
+    enable_service => $enable_subscriber_list_update_queue_consumers,
+    process_type   => 'subscriber-list-details-update-major-consumer',
+    process_regex  => '\/rake message_queues:subscriber_list_details_update_major_consumer',
+  }
+
+  govuk::procfile::worker { 'email-alert-service-subscriber-list-details-update-major-consumer':
```

Comment:
Should be `email-alert-service-subscriber-list-details-update-minor-consumer`

Reply:
urgh that got lost in a rebase, sorry 1 sec